llarp/context.cpp

```diff
@@ -60,20 +60,11 @@ namespace llarp
     }
 
     // Router config
-    if(!singleThreaded && config->router.workerThreads > 0 && !worker)
-    {
-      worker.reset(
-          llarp_init_threadpool(config->router.workerThreads, "llarp-worker"));
-    }
+    if(config->router.workerThreads <= 0)
```

Comment:
shouldn't this be addressed wherever the config is loaded and thus not a valid value here?  Defensive coding is good practice, of course, just want to make sure the config isn't silently "okay".

Reply:
this should be in both config load and apply